RRE_measurements.py

```diff
@@ -0,0 +1,101 @@
+''' Definition of the measurement class specific to range, range rate, and elevation measurements
+
+Parent class meas records a measurement to be fed into the Kalman Filter
+
+All children classes of meas contain any extraneous data and the system dynamics
+
+station class is used for range, rangerate measurements taken from AWS groundstation, defines
+a station (latitude and longitude)
+
+Author:Connor Johnstone
+'''
+
+#Import Libraries
+import numpy as np
+import WheresMySat as WMS
+from measurements import Meas
+import astropy.coordinates as astro
+from astropy import units as u
+from astropy import time
+
+
+class RangeMeas(Meas):
+    ''' Range Measurement
+    
+    Includes information common to all measurements plus addition info (station_id) specific to
+    Range measurements
+    
+    Arguments:
+        - station_id: (station) Station object from Amazon groundstation (may be expanded in the
+        future) which includes latitude, longitude of station
+    '''
+    def __init__(self,value,time,mission_id,station_id):
+        self.range_meas = value
```

Comment:
do we need to duplicate range_meas and value? Can just access self.value once you've used super()